br/pkg/task/restore_data.go

```diff
@@ -146,28 +145,43 @@ func RunResolveKvData(c context.Context, g glue.Glue, cmdName string, cfg *Resto
 	}()
 
 	var allStores []*metapb.Store
-	allStores, err = conn.GetAllTiKVStoresWithRetry(ctx, mgr.GetPDClient(), util.SkipTiFlash)
+
+	var numOnlineStore int
+	err = utils.WithRetry(
```

Comment:
why add another layer of retry on top of GetAllTiKVStoresWithRetry

Reply:
the inner function has retry only 500 ms, it is enough for BR other restore, but here for EBS, 2nd restore may failure since pd + tikv startup take longer than 500 ms, to separate from BR legacy restore, add XX Ext retry.